peepholelib/classifier/classifier_base.py

```diff
@@ -59,7 +54,22 @@ def __init__(self, **kwargs):
         # computer in compute_empirical_posteriors()
         self._empp = None
 
+        # defined in save() or load()
+        self.file_path = None
```

Comment:
the `file_path` is fixed depending on `nl_class` and `nl_model`, so we do not need the `self.file_path` variable here.